### PR TITLE
docs: add HPC Launcher job exit code known issue

### PR DESCRIPTION
**Pull request by CanmingCobble**
## Description

image

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-known-issues.rst [hunk 1]
```diff
@@ -17,6 +17,14 @@ ignored.
 -  ``resources.priority``
 -  ``resources.weight``
 
+******************************************
+ Slurm and PBS Job Exit Code Report Issue
```
Comment (jerryharrow) [inline docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-known-issues.rst:21]: IMO would be better to add to the existing "Slurm Known Issues" section, and duplicate in "PBS Known Issues"

Generally, we don't have per-issue headings.

Comment (CanmingCobble) [reply docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-known-issues.rst:21]: There is a section  "Slurm Known Issues" and there is a section  "Additional Known issues". What about adding it to the "Additional Known issues" without creating a new section "PBS Known issues"?

Comment (jerryharrow) [reply docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-known-issues.rst:21]: That works for me also.

Comment (CanmingCobble) [reply docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-known-issues.rst:21]: Had trouble merging up to the master so created a new branch and new PR https://github.com/determined-ai/determined/pull/6233

**Pull request closed without merging**